### PR TITLE
Use requests for polling VTN

### PR DESCRIPTION
**Pull request by rafalgajda-silvair**
Rework to use requests library

### CODE REVIEW

File: oadr2/poll.py [hunk 1]
```diff
@@ -179,23 +140,26 @@ def query_vtn(self):
...
 
-        if CONTENT_TYPE not in resp.headers.get("Content-Type", ""):
-            logging.warn('Unexpected content type')
+        try:
+            resp = requests.post(
+                event_uri,
+                cert=(
+                    self.ven_client_cert_pem,
+                    self.ven_client_cert_key
+                ) if self.ven_client_cert_pem and self.ven_client_cert_key else None,
```
Comment (bartoszflis-silvair) [inline oadr2/poll.py]: How about pulling it outside of the argument list?

File: oadr2/poll.py [hunk 2]
```diff
@@ -223,106 +187,15 @@ def send_reply(self, payload, uri):
...
-            self._tunnel()
-
-        if self.ca_certs and not os.path.isfile(self.ca_certs):
-            logging.warn("CA certs file does not exist: %s", self.ca_certs)
+        resp = requests.post(
+            uri,
+            cert=(
+                self.ven_client_cert_pem,
+                self.ven_client_cert_key
+            ) if self.ven_client_cert_pem and self.ven_client_cert_key else None,
```
Comment (bartoszflis-silvair) [inline oadr2/poll.py]: How about pulling it outside of the argument list?

Comment (rafalgajda-silvair) [reply oadr2/poll.py]: Changed

**Review by bartoszflis-silvair — APPROVED** [review]

File: requirements.txt [hunk 3]
```diff
@@ -4,3 +4,6 @@ python-dateutil==2.1
 six==1.12.0
 sleekxmpp==1.1.11
 freezegun==0.3.15
+
+requests~=2.24.0
```
Comment (michallowasrzechonek-silvair) [inline requirements.txt:7]: Add this to `setup.py` `install_requires` as well

File: requirements.txt [hunk 4]
```diff
@@ -4,3 +4,6 @@ python-dateutil==2.1
...
+setuptools~=47.2.0
```
Comment (michallowasrzechonek-silvair) [inline requirements.txt]: Why `setuptools`?

**Review by michallowasrzechonek-silvair — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 4 above)
Comment (rafalgajda-silvair) [reply requirements.txt]: That was added automatically by IDE. I'll remove it.

File: requirements.txt (hunk 3 above)
Comment (rafalgajda-silvair) [reply requirements.txt:7]: Done.

**Review by michallowasrzechonek-silvair — APPROVED** [review]

**Pull request merged**